pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/converters/SampleTypeConverter.java

```diff
@@ -0,0 +1,132 @@
+package ca.on.oicr.pinery.lims.miso.converters;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SampleTypeConverter {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)